Brewfile-core

```diff
@@ -0,0 +1,44 @@
+tap 'cloudfoundry/tap'
+tap 'k14s/tap'
+tap 'pivotal/tap'
+tap 'buildpack/tap'
+
+# Core utilities
+brew 'git'
+brew 'lastpass-cli'
+brew 'wget'
+brew 'coreutils'
+brew 'direnv'
```

Comment:
bash and bash-completion should be in here. no one should have to care about whatever random ancient version of bash OSX comes with, and we shouldn't make people who can't install the monolithic thing to be typing every letter of every command on the command line.

Reply:
OS X no longer deploys bash as the default shell.

I thought about it, and it helps to read this comment. I'll add it back, if only for the fact that on _some_ peoples machines, are very old and insecure versions of bash.